kolibri/utils/cli.py

```diff
@@ -636,6 +634,8 @@ def main(args=None):  # noqa: max-complexity=13
         return
 
     if arguments['start']:
+        # Check if the content directory exists when Kolibri runs after the first time.
+        _first_run()
```

Comment:
Ah, sorry, I have been giving ill-advice, will do a commit directly to your branch to fix it!

Reply:
thank you for the commit!